theano/tensor/nnet/abstract_conv.py

```diff
@@ -1613,7 +1612,7 @@ def perform(self, node, inp, out_):
         if isinstance(mode, tuple):
             pad = tuple(int(mode[i]) for i in range(self.convdim))
             mode = "valid"
-            new_img = numpy.zeros((img.shape[0], img.shape[1]) +
+            new_img = np.zeros((img.shape[0], img.shape[1]) +
                                   tuple(img.shape[i + 2] + 2 * pad[i]
```

Comment:
You need to dedent all following lines up to the closing bracket by the number of characters you removed (3). Same for the other `np.zeros` instances. Otherwise PEP-0008 is violated.